gym_reachability/gym_reachability/envs/one_player_reach_avoid_lunar_lander.py

```diff
@@ -484,10 +595,8 @@ def visualize(
       for x_ii, x_dot in enumerate(self.slices_x):
         ax = self.axes[y_jj][x_ii]
         ax.cla()
-        v, xs, ys = self.get_value(
-            q_func, nx, ny, x_dot=x_dot, y_dot=y_dot, theta=0, theta_dot=0,
-            addBias=addBias
-        )
+        v1, xs = self.get_value_x(q_func, nx, ny, x_dot=x_dot, y_dot=0, theta=0, theta_dot=0, addBias=addBias)
+        v2, ys = self.get_value_y(q_func, nx, ny, x_dot=0, y_dot=y_dot, theta=0, theta_dot=0, addBias=addBias)
```

Comment:
add if statement that checks whether we are looking for decomposition or just the regular algorithm.

this should be in the `sim_lunar_lander.py` config